README.md

```diff
@@ -8,17 +8,18 @@ This extension adds support for [Racket](http://www.racket-lang.org) to VS Code.
 2. Make sure you have `raco` on your path (see section Troubleshooting)
 3. Install the [racket-langserver](https://github.com/jeapostrophe/racket-langserver) by running the following command in the terminal:
 
-    ```bash
-    raco pkg install racket-langserver
-    ```
+   ```bash
```

Comment:
Hmm, not entirely why these lines have changed...? Please revert any changes that aren't related to the macro stepper.

Reply:
Oops, I guess that's my markdown formatter, I will revert them.